frontend/src/features/public-form/components/StatusTrackerPage/StatusTrackerPage.tsx

```diff
@@ -0,0 +1,174 @@
+export const StatusTrackerBaseGridLayout = (props: GridProps) => (
+  <AppGrid
+    templateRows={{ md: 'auto 1fr auto', lg: '1fr auto' }}
+    alignItems="start"
+    {...props}
+  />
+)
+
+const StatusTrackerFormInfo = (): JSX.Element => {
+  // trying to get logo to work
+  const { captchaContainerId, form } = usePublicFormContext()
+
+  return (
+    <Box w="100%">
+      {/* Hiding recaptcha container here as id needs to be grabbed within publicFormProvider and hidden from form footer */}
+      <Box id={captchaContainerId} display="none" mt="2rem" />
+      <Flex direction="row" alignItems="center" gap="0.5rem">
+        <Text textStyle="body-2" color="#FFFFFF">
+          {form?.title}
+        </Text>
+      </Flex>
+      <Text textStyle="h6" color="#FFFFFF" mt="1rem">
+        Track your FormSG Response status
+      </Text>
+    </Box>
+  )
+}
+
+export const StatusTrackerPage = (): JSX.Element => {
+  const { formId, submissionId } = useParams()
+  if (!formId) throw new Error('No formId provided')
+  if (!submissionId) throw new Error('No submissionId provided')
+
+  const { data, isLoading, error } = useStatusTracker(submissionId)
+
+  if (isLoading) return <StatusTrackerSkeletonPage />
+
+  if (error || !data || !data.submittedSteps || !data.workflow)
+    return <NotFoundErrorPage />
+
+  const { submittedSteps, workflow } = data
+
+  const stepData: StepData[] = workflow.map((step, index) => {
+    const name = step.step_name ? step.step_name : `Step ${index + 1}`
+    const stepNumber = index + 1
+
+    const submittedStep = submittedSteps[index]
+
+    const workflowStatus =
+      index < submittedSteps.length
+        ? workflow[index].approval_field // if this step is an approval step
+          ? 'status' in submittedStep // if status is in submitted step
+            ? submittedStep.status
+            : WorkflowStatus.COMPLETED
+          : WorkflowStatus.COMPLETED
+        : WorkflowStatus.PENDING
+
+    if (index < submittedSteps.length) {
+      return {
+        name: name,
+        stepNumber: stepNumber,
+        timestamp: submittedSteps[index].submittedAt,
+        workflowStatus: workflowStatus,
+      }
+    }
+
+    return {
+      name: name,
+      stepNumber: stepNumber,
+      workflowStatus: workflowStatus,
+      isCurrentPendingStep: index == submittedSteps.length,
+    }
+  })
+
+  const startTime = Date.now()
+  return (
+    <PublicFormProvider formId={formId} startTime={startTime}>
+      <BackgroundBox>
+        <StatusTrackerBaseGridLayout flex={1}>
+          <StatusTrackerFormInfoGridArea>
+            <StatusTrackerFormInfo />
+          </StatusTrackerFormInfoGridArea>
+          <TimelineGridArea>
+            <Box mt={{ base: '1rem', lg: '2.33rem' }}>
+              <Flex direction="column" align="flex-start">
+                {/* manually transforming horizontal positioning left because publicFormLogo has column layout */}
+                <Box transform="translateX(-1.5rem)">
```

Comment:
Possible to use negative margins to correct the additional paddings in logo.
```suggestion
                <Box ml="-1.5rem">
```